src/Console.cc

```diff
@@ -30,12 +30,11 @@ using namespace common;
 FileLogger ignition::common::Console::log("");
 
 #ifdef _WIN32
```

Comment:
It looks like this `ifdef` isn't needed anymore

Reply:
Removed, but added warning that it is not a rule that the values are the same on win and linux.